readme.md

```diff
@@ -23,14 +23,13 @@ You've written the following HTML. When you look at it in your browser, it's jus
   </body>
 </html>
 ```
-
-> Your answer...
+It's missing a closing title tag.
 
 ## Question 2
 
 What's the purpose of the `alt` attribute on image tags?
 
-> Your answer...
+To display the name of the image if it doesn't display properly.
```

Comment:
It defines the text that should be communicated if the user can't see the image. This could be because they are blind or because the image link is broken.